frontend/translations/en.po

```diff
@@ -6577,7 +6577,7 @@ msgstr "Add a theme (i.e. Light)"
 
 #: src/app/main/ui/workspace/tokens/modals/themes.cljs:56
 msgid "workspace.token.new-theme"
-msgstr "New theme"
+msgstr "Add new theme"
```

Comment:
There is a problem with editing an old translation. 
This string was created with a sentence “new-theme” and it was related to the id “workspace.token.new-theme”.
And users could start translating it, in fact this string has already been translated to 10 languages. 
Now you change the string, but you don't change the id, so those 10 lang are going to be translated with the old meaning. 
To avoid this you must create a new pair id - tr